develdocs/doctum.php

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/php
+<?php
+$_CLI = getopt('', array('slug:', 'root:', 'build-dir:', 'cache-dir:', 'output-config:', 'title::', 'title-of-composer'));
+
+$SOURCE = 'src';
+
+if (is_dir($_CLI['root'] . '/src') === false) {
+    $SOURCE = 'libraries';
+}
+
+$_CLI['root'] = rtrim($_CLI['root'], '/') . '/';
+
+$output = '<?php
```

Comment:
I think it's better to use Nowdoc string with sprintf for the variables.

Reply:
Good idea, done :)